test/com/esotericsoftware/kryo/serializers/FieldSerializerTest.java

```diff
@@ -19,7 +19,24 @@
 
 package com.esotericsoftware.kryo.serializers;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.io.ByteArrayOutputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import org.junit.Test;
+import org.objenesis.strategy.StdInstantiatorStrategy;
```

Comment:
Your import order is still wrong for some files. Please apply the `.importorder` settings to those files.

Reply:
Fixed

Reply:
Thanks!